pkg/registry/minion/rest.go

```diff
@@ -91,7 +91,18 @@ func (rs *REST) Get(ctx api.Context, id string) (runtime.Object, error) {
 
 // List satisfies the RESTStorage interface.
 func (rs *REST) List(ctx api.Context, label, field labels.Selector) (runtime.Object, error) {
-	return rs.registry.ListMinions(ctx)
+	nodes, err := rs.registry.ListMinions(ctx)
```

Comment:
@kazegusuri You've started to port the minion/node registry to generic already? Or would you like this PR to be merged first?